src/Forms/GridField/GridFieldDetailForm_ItemRequest.php

```diff
@@ -281,34 +281,58 @@ protected function getRightGroupField()
         $rightGroup->setFieldHolderTemplate(get_class($rightGroup) . '_holder_buttongroup');
 
         $previousAndNextGroup = CompositeField::create()->setName('PreviousAndNextGroup');
-        $previousAndNextGroup->addExtraClass('circular-group mr-2');
-        $previousAndNextGroup->setFieldHolderTemplate(get_class($previousAndNextGroup) . '_holder_buttongroup');
+        $previousAndNextGroup->addExtraClass('btn-group--circular mr-2');
+        $previousAndNextGroup->setFieldHolderTemplate(CompositeField::class . '_holder_buttongroup');
```

Comment:
This would've resulted in a fatal error (template not found) if CompositeField was rerouted with Injector to MyCompositeField